ui/app/controllers/pipeline.js

```diff
@@ -25,6 +31,12 @@ export default class PipelineController extends Controller {
 
   @action
   async stopPipeline(pipeline) {
+    try {
+      await pipeline.startPipeline();
```

Comment:
Is there also a `stopPipeline` method that could be used here, instead of `startPipeline`?